ibis/expr/datatypes.py

```diff
@@ -390,78 +415,79 @@ def valid_literal(self, value):
 
 class Int8(Integer):
 
+    __slots__ = ()
```

Comment:
Do `__slots__` inherit in any reasonable way? (I assume not)

Reply:
That's right. Stated here: https://docs.python.org/3.6/reference/datamodel.html#notes-on-using-slots
> The action of a `__slots__` declaration is limited to the class where it is defined. As a result, subclasses will have a `__dict__` unless they also define `__slots__` (which must only contain names of any additional slots).